minimal/src/Components/Home.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+import { Text, View, StyleSheet } from 'react-native'
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+import InputName from './InputName';
+
+export class Home extends Component {
```

Comment:
Because this component only has a `render` method, please convert it to a functional component.